src/pages/resultsView/survival/SurvivalTab.tsx

```diff
@@ -0,0 +1,100 @@
+            this.props.store.diseaseFreeUnalteredPatientSurvivals.isPending) {
+            return <Loader isLoading={true}/>;
+        }
+
+        let content:any = [];
+
+        if (this.props.store.overallAlteredPatientSurvivals.isComplete &&
+            this.props.store.overallUnalteredPatientSurvivals.isComplete &&
+            this.props.store.overallAlteredPatientSurvivals.result.length > 0 &&
+            this.props.store.overallUnalteredPatientSurvivals.result.length > 0) {
+            content.push(
+                <SurvivalChart alteredPatientSurvivals={this.props.store.overallAlteredPatientSurvivals.result}
+                               unalteredPatientSurvivals={this.props.store.overallUnalteredPatientSurvivals.result}
+                               title={this.overallSurvivalTitleText}
+                               xAxisLabel="Months Survival"
+                               yAxisLabel="Surviving"
+                               totalCasesHeader="#total cases"
+                               statusCasesHeader="#cases deceased"
+                               medianMonthsHeader="median months survival"
+                               yLabelTooltip="Survival estimate"
+                               xLabelWithEventTooltip="Time of death"
+                               xLabelWithoutEventTooltip="Time of last observation"
+                               fileName="Overall_Survival"/>);
+        } else {
+            content.push(<div className={styles.NotAvailable}>{this.overallSurvivalTitleText} not available</div>);
+        }
+
+        if (this.props.store.diseaseFreeAlteredPatientSurvivals.isComplete &&
+            this.props.store.diseaseFreeUnalteredPatientSurvivals.isComplete &&
+            this.props.store.diseaseFreeAlteredPatientSurvivals.result.length > 0 &&
+            this.props.store.diseaseFreeUnalteredPatientSurvivals.result.length > 0) {
+            content.push(
+                <SurvivalChart alteredPatientSurvivals={this.props.store.diseaseFreeAlteredPatientSurvivals.result}
+                           unalteredPatientSurvivals={this.props.store.diseaseFreeUnalteredPatientSurvivals.result}
+                           title={this.diseaseFreeSurvivalTitleText}
+                           xAxisLabel="Months Disease Free"
+                           yAxisLabel="Disease Free"
+                           totalCasesHeader="#total cases"
+                           statusCasesHeader="#cases relapsed"
+                           medianMonthsHeader="median months disease free"
+                           yLabelTooltip="Disease free estimate"
+                           xLabelWithEventTooltip="Time of relapse"
+                           xLabelWithoutEventTooltip="Time of last observation"
+                           fileName="Disease_Free_Survival"/>);
+        } else {
+            content.push(<div className={styles.NotAvailable}>{this.diseaseFreeSurvivalTitleText} not available</div>);
```

Comment:
seems like this would read non available when in pending state

Reply:
Not sure what you mean, but we can discuss later, can merge for now.

Reply:
@alisman There is
```
if (this.props.store.overallAlteredPatientSurvivals.isPending ||
            this.props.store.overallUnalteredPatientSurvivals.isPending ||
            this.props.store.diseaseFreeAlteredPatientSurvivals.isPending ||
            this.props.store.diseaseFreeUnalteredPatientSurvivals.isPending) {
            return <Loader isLoading={true}/>;
        }
```
on the top.